WORKSPACE

```diff
@@ -76,6 +76,14 @@ docker_pull(
     repository = "k8s-prow/git",
 )
 
+docker_pull(
+    name = "python",
+    digest = "sha256:3f60d995e4aa7d7a5e4783c185e13d53e124ef0ad87bc1492f869cbeead331ee",
+    registry = "index.docker.io",
+    repository = "library/python",
+    tag = "2.7.14-alpine3.7",
```

Comment:
does this version of the python image contain git?

Reply:
nope :cry: good catch